src/Microsoft.CodeAnalysis.Analyzers/UnitTests/InternalImplementationOnlyTests.cs

```diff
@@ -517,265 +517,249 @@ public async Task Basic_VerifyISymbol()
         {
             var source = @"
 ' Causes many compile errors, because not all members are implemented.
-Class Foo 
+Class C1
     Implements Microsoft.CodeAnalysis.ISymbol
 End Class
-Class Bar
+Class C2
```

Comment:
Using `SomeClass` and `SomeOtherClass` here was causing the test to hang... I couldn't find out why (I didn't spent too long debugging it).